src/markdown_command.egg-info/PKG-INFO

```diff
@@ -0,0 +1,29 @@
+Metadata-Version: 2.1
```

Comment:
I don't think this should be here. I would remove files inside `*.egg-info` directory from the **repository** (they should stay in your local env) and add `*.egg-info` to the `.gitignore`. These files are always generated from source, so I don't think we need them nor I ever saw them in any OS Python repository.